client/mapctrl.cpp

```diff
@@ -316,7 +325,10 @@ void map_view::shortcut_pressed(shortcut_id id)
     break;
 
   case SC_MAP_EDITOR:
-    queen()->map_editor_wdg->check_open();
+    // Editor is not meant to be used with observers
```

Comment:
Why?

Reply:
Why is this a problem? The legacy fc editor operates the same. FWIW

https://github.com/longturn/freeciv21/blob/master/client/editor/map_editor.cpp#L169

barfs bad if you are an observer (of any kind) when invoked. So that is the main reason.

Reply:
Ok, let's not try to outguess the old editor ^^